pom.xml

```diff
@@ -52,6 +52,15 @@
 			<version>RELEASE</version>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-webflux</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>mysql</groupId>
+			<artifactId>mysql-connector-java</artifactId>
+			<version>8.0.33</version>
+		</dependency>
```

Comment:
I think this dependency isn't required in this project due to the fact that you have spring-boot-starter-data-jpa dependency. But let me know if I'm wrong :)

Reply:
I want to have MySQL database for the test part of my application, therefore I need two dependencies. 
1. `The spring-boot-starter-data-jpa` is a starter for using Spring Data JPA with Hibernate. It simplifies the integration of Spring JPA/Hibernate with my project
2. `mysql-connector-java` is a JDBC driver that provides database connectivity through JDBC. This is needed for communication with MySQL db.

So I need both, `the spring-boot-starter-data-jpa` dependency for simplifying database interaction, and the `mysql-connector-java` dependency for connecting application to my MySQL database

Reply:
But have you tried to run the application without the connector dependency? It looks like it works on my end without it.

Reply:
I think it works without this dependency. We have to check it on our classes, cause I have one more issue with that! :)

Reply:
OK, so it turns out this dependency is needed :)